src/CommandList.java

```diff
@@ -0,0 +1,20 @@
+import java.util.HashMap;
+import java.util.Map;
+public class CommandList {
```

Comment:
Why you need this object ? What different between Commands ?

Reply:
I renamed CommandList to Commands.
But on my github, my files were not deleted :)